src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -3,242 +3,214 @@ package edu.berkeley.ce.sparkrocks
+    * @param jointSets Seq of Seq's of Joints representing input joint sets
+    * @param inputVolume Block representing input rock volume
+    * @param numPartitions Number of partitions desired
+    * @param jointSetSpan Number of joint sets to search at a time for partition generation
+    * @param spanCount Number or joint sets to search for seed joints
+    * @param seedJoints Seq of joints that were not used in seed block generation
+    * @param remainingJoints Seq of joints that were not selected seed joints
+    * @return Returns a tuple containing two Seq's of Joints. The first Seq contains the seed joints that
+    *         best divide the input volume. The second Seq contains the joints that remain from the joint
+    *         sets that have been checked.
     */
   @tailrec
-  private def findSeedJoints(jointSet: Seq[Joint], rockVolume: Block, numProcessors: Integer, tolerance: Double = 0.01,
-                             stepSize: Double = 0.01): Option[Seq[Joint]] = {
-    val totalVolume = rockVolume.volume
-    val volumePerProc = rockVolume.volume / numProcessors
-    val seedJoints = selectSeedJoints(jointSet, rockVolume, volumePerProc, totalVolume,
-      Seq.empty[Joint], tolerance, numProcessors)
-
-    if (stepSize < 0.01) {
-      // Unable to find satisfactory seed joints using current joint set
-      None
-    } else if (seedJoints.length != numProcessors - 1) {
-      val newStepSize = if (tolerance >= 1.0) { stepSize / 2.0 } else { stepSize }
-      val newTolerance = if (tolerance >= 1.0) { newStepSize } else { tolerance + stepSize }
-      findSeedJoints(jointSet, rockVolume, numProcessors, newTolerance, newStepSize)
+  def searchJointSets(jointSets: Seq[Seq[Joint]], inputVolume: Block, numPartitions: Int, jointSetSpan: Int,
+                      spanCount: Int = 1, seedJoints: Seq[Joint] = Seq.empty[Joint],
+                      remainingJoints: Seq[Joint] = Seq.empty[Joint]): (Seq[Joint], Seq[Joint]) = {
+    // Checked specified span of joint sets, return
+    if (spanCount > jointSetSpan) {
+      (seedJoints, remainingJoints)
     } else {
-      val initialBlock = Seq(rockVolume)
-      val blocks = seedJoints.foldLeft(initialBlock) { (currentBlocks, joint) =>
-        currentBlocks.flatMap(_.cut(joint))
-      }
-      val nonRedundantBlocks = blocks map { case block @ Block(blockCenter, _, _) =>
-          Block(blockCenter, block.nonRedundantFaces)
-      }
-      val volumes = nonRedundantBlocks.map(_.volume)
-      val avgVolume = volumes.sum / volumes.length
-      println("\nLOAD BALANCE INFORMATION:")
-      println(s"Average Volume: %.2f".format(avgVolume))
-      println(s"Max Volume: %.2f".format(volumes.max))
-      println(s"Min Volume: %.2f".format(volumes.min))
-      Some(seedJoints)
+      val totalVolume = inputVolume.volume
+      // Determine remaining partitions that need to be found. It is not possible to know before hand how many
```

Comment:
This comment makes sense. Since it occupies so many lines, can you use the multi-line instead? Something like this:
```scala
/*
 * this is a comment
 * on multiple lines
 */
```